### PR TITLE
Use voting validations for proposals

### PR DESCRIPTION
**Pull request by ChaituVR**
Related to https://github.com/snapshot-labs/snapshot/issues/3466

### CODE REVIEW

**Comment by samuveth** [comment]
Is this PR ready for review?

File: src/writer/proposal.ts [hunk 1]
```diff
@@ -65,19 +65,39 @@ export async function verify(body): Promise<any> {
...
+          body.address,
+          space.id,
+          space.network,
+          'latest',
+          validationParams,
+          {}
+        );
+      }
+
+      if (isValid === false) return Promise.reject('validation failed');
```
Comment (ChaituVR) [inline src/writer/proposal.ts:94]:
```suggestion
      if (!isValid) return Promise.reject('validation failed');
```

File: src/writer/proposal.ts [hunk 2]
```diff
@@ -65,19 +65,39 @@ export async function verify(body): Promise<any> {
...
     try {
       const validationName = space.validation?.name || 'basic';
       const validationParams = space.validation?.params || {};
-      const isValid = await snapshot.utils.validations[validationName](
-        body.address,
-        space,
-        msg.payload,
-        validationParams
-      );
+      const minScore = space.validation?.params?.minScore || space.filters.minScore;
```
Comment (bonustrack) [inline src/writer/proposal.ts:72]: Filters may not exist here
```suggestion
      const minScore = space.validation?.params?.minScore || space.filters?.minScore;
```

**Review by bonustrack — APPROVED** [review]
utACK

**Pull request merged**